packages/trader/src/App/Components/Elements/Media/media-icon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type TMediaIcon = {
+    disabled: React.SVGAttributes<SVGElement>;
+    enabled: React.SVGAttributes<SVGElement>;
+    id: string;
+    is_enabled: boolean;
+};
+const MediaIcon = ({ id, is_enabled, enabled, disabled }: TMediaIcon) => {
+    const Icon = (is_enabled ? enabled : disabled) as React.ElementType;
```

Comment:
```suggestion
    const Icon: React.ElementType = (is_enabled ? enabled : disabled);
```
Does this work? To avoid type casting with `as`

Reply:
Done, thank you